src/web/extractors.rs

```diff
@@ -1227,13 +1227,6 @@ impl ToString for Offset {
 impl FromStr for Offset {
     type Err = ParseIntError;
     fn from_str(s: &str) -> Result<Self, Self::Err> {
-        // issue559: Disable ':' support for now: simply parse as i64 as
```

Comment:
This is going to turn it on for spanner as well. Can you make turning it on a configuration parameter?

Reply:
Default it to off

Reply:
This method cracks the Offset correctly, based on what it's handed. Spanner (at least currently) never reads (or writes) the timestamp. It's unlikely that the data store would switch while processing, but even if it did, I believe things would continue to work correctly. I will add a bit of logic to the mysql side to use the numeric offset if the sorting is by time and there's no timestamp specified, just to cover a possible edge case.